pinot-filesystem/src/main/java/org/apache/pinot/filesystem/PinotFS.java

```diff
@@ -95,7 +95,7 @@ public abstract boolean exists(URI fileUri)
       throws IOException;
 
   /**
-   * Returns the length of the file at the provided location. Will throw exception if a directory
+   * Returns the length of the file at the provided location. Will throw exception if a directory.
```

Comment:
Looks like this API was added to this class only for testing? Pinot code does not seem access it.